mkosi

```diff
@@ -1898,7 +1900,12 @@ def install_opensuse(args, workspace, *, run_build_script):
             cmdlinefile.write(args.kernel_commandline + " root=/dev/gpt-auto-root\n")
 
 
-def install_distribution(args, workspace, *, cached, **kwargs):
+def install_distribution(args: CommandLineArguments,
+                         workspace: str,
+                         *,
+                         run_build_script: bool,
```

Comment:
Is this parameter new? I don’t see it used below.

Reply:
Whoops, it was previously part of `**kwargs`.  A later commit fixes this when it drops kwargs.